pkg/storage/tableland/impl/tableland/store.go

```diff
@@ -73,92 +73,105 @@ func (s *Store) InsertParts(ctx context.Context, parts []local.Part) error {
 }
 
 // InsertLayers implements InsertLayers.
-func (s *Store) InsertLayers(ctx context.Context, cid string, layers []local.Layer) error {
+func (s *Store) InsertLayers(ctx context.Context, layers []local.Layer) error {
```

Comment:
Same thing, imagery cid now stored in the lookups table, so not needed in this file anymore. Plus some cleanup.